src/System.ComponentModel.EventBasedAsync/src/System/ComponentModel/BackgroundWorker.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace System.ComponentModel
 {
-    public class BackgroundWorker : IDisposable
+    public class BackgroundWorker : System.ComponentModel.Component
```

Comment:
IIRC, because Component wasn't a base class, we added finalizers to some of these derived types like BackgroundWorker, so that if types derived from BackgroundWorker and override Dispose(bool), they would still have their Dispose method called on finalization (since Component has one).  Now that we're adding back Component, we should remove such finalizers we added just to core.  In addition to being unnecessary, they could actually lead to problems, if they result in Dispose(false) being executed twice for an instance.